pkg/automationconfig/automation_config_builder.go

```diff
@@ -17,6 +17,7 @@ const (
 	ReplicaSetTopology    Topology = "ReplicaSet"
 	maxVotingMembers      int      = 7
 	arbitersStartingIndex int      = 100
+	defaultPort           int      = 27017
```

Comment:
The `defaultPort` is also declared at `mongodbcommunity_types.go`, is it possible to consolidate it to only one place?

Reply:
We have been trying to avoid using `util` package for a while now, basically, we are moving out of the util package in enterprise since it's very unclear what those collections of packages entail. 

I still this this should be declared at one place to avoid any potential bug later, but will leave it to you to decided the preferred location.

Reply:
Just noticed we already import `automationconfig` package in `mongodbcommunity_types.go`, so maybe we can reuse this constant in mongodbcommunity_types.go ?